provider/aci.go

```diff
@@ -61,6 +61,10 @@ const (
 	gpuTypeAnnotation = "virtual-kubelet.io/gpu-type"
 )
 
+const (
+	priorityTypeAnnotation = "priority"
```

Comment:
This string should be prefixed in some way in order to not conflict with potential annotations already in use by a customer pod for maximum compatibility with customers' existing pods. I suspect `priority` is a reasonably frequent "private" annotation. 

See for example `virtualKubeletDNSNameLabel` or `gpuTypeAnnotation`. 

I'd say that an argument can be made that given how ACI-specific this is, we could "namespace" it even more specifically than just virtual kubelet, but at the very least, IMO, it should match the other annotation-based extensions here.